### PR TITLE
Added Email Verification support

### PR DESCRIPTION
**Pull request by nithinsubhash**
Added support for `v2/verification/email` support.

#### New methods
- `fullcontact_client.verification.email()`
- `fullcontact_client.verification.email_async()`

### CODE REVIEW

File: tests/test_email_verification.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@pytest.fixture
+def mock_good_response(monkeypatch):
+    def mock_post(*args, **kwargs):
+        return MockResponse.get_mock_response(
+            REQUEST_TYPE,
+            method=METHOD_EMAIL,
+            test_scenario=SCENARIO_POSITIVE
+        )
+
+    monkeypatch.setattr(Session, "get", mock_post)
```
Comment (nathan-pr) [inline tests/test_email_verification.py]: This is a post but really a get?

File: fullcontact/api/verification_api.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+class VerificationApi(BaseApi):
+    _BASE_URL = "https://api.fullcontact.com/v2/"
+
+    _email_verification_endpoint = "verification/email"
+    _email_verification_request = EmailVerificationRequestSchema()
+    _email_verification_response = EmailVerificationResponse
+
+    def email(self, email: str, headers: dict = None) -> _email_verification_response:
+        r"""
+        POST the email to FullContact email verification API
```
Comment (nathan-pr) [inline fullcontact/api/verification_api.py]: This and the call below is really a GET right?

**Review by nathan-pr — APPROVED** [review]
Approved but with 2 small questions around GET/POST

File: tests/test_email_verification.py (hunk 1 above)
Comment (nithinsubhash) [reply tests/test_email_verification.py]: An error while copying. Fixed!

File: fullcontact/api/verification_api.py (hunk 2 above)
Comment (nithinsubhash) [reply fullcontact/api/verification_api.py]: Yes, it is a GET. An error while copying. Fixed!

**Pull request merged**